hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -306,6 +307,33 @@ public static Long getFolderByNameForUser(String name, Long parentId, Long userI
         return childrenFolders;
     }
 
+
+
+    public static List<Tuple> getMapsForUser(Users user) {
+
+        SQLQuery<Tuple> q = createQuery()
+                .select(maps, folders.id, folders.publicCol)
+                .from(maps)
+                .leftJoin(folderMapMappings).on(folderMapMappings.mapId.eq(maps.id))
+                .leftJoin(folders).on(folders.id.eq(folderMapMappings.folderId))
+                .orderBy(maps.displayName.asc());
+        if(user != null) {
```

Comment:
we can combine the inner if statement condition with this one

`if(user != null && !UserResource.adminUserCheck(user))`

Reply:
Actually I moved this up to short circuit the query so that a null user causes an empty list to be returned.

Reply:
i like that more!